build/Dockerfile

```diff
@@ -10,6 +10,8 @@ COPY _defaults.yml ${HOME}/_defaults.yml
 COPY deploy/crds ${HOME}/deploy/crds
 COPY manifests/generated ${HOME}/manifests/generated
 COPY build/csv-generator.sh /usr/bin/csv-generator
+#original module k8s_status contains bug, which prevents ssp operator to correctly set conditions
+COPY build/k8s_status.py /usr/share/ansible/openshift
```

Comment:
please mention the u/s k8s_status.py bug/issue so we can track it and drop this workaround as soon as possible